packages/web-app/src/containers/configureAddMember/index.tsx

```diff
@@ -0,0 +1,352 @@
+        tokenSymbol,
+    ]);
+
+    /*************************************************
+     *             Callbacks and Handlers            *
+     *************************************************/
+
+    // clear field when there is a value, else paste
+    const handleAdornmentClick = useCallback(
+        (value: string, onChange: (value: string) => void) => {
+            // when there is a value clear it
+            if (value) {
+                onChange('');
+                alert(t('alert.chip.inputCleared'));
+            } else handleClipboardActions(value, onChange, alert);
+        },
+        [alert, t]
+    );
+
+    const resolveEnsNameFromAddress = useCallback(
+        (address: string | Promise<string>) => provider.lookupAddress(address),
+        [provider]
+    );
+
+    const resolveAddressFromEnsName = useCallback(
+        (ensName: string | Promise<string>) => provider.resolveName(ensName),
+        [provider]
+    );
+
+    const handleValueChanged = useCallback(
+        (value: InputValue, onChange: (...event: unknown[]) => void) =>
+            onChange(value),
+        []
+    );
+
+    /*************************************************
+     *                Field Validators               *
+     *************************************************/
+    const addressValidator = useCallback(
+        async (address: string) => {
+            if (isNativeToken(address)) return true;
+
+            const validationResult = await validateTokenAddress(address, provider);
+
+            // address invalid, reset token fields
+            if (validationResult !== true) {
+                resetField(`actions.${actionIndex}.tokenName`);
+                resetField(`actions.${actionIndex}.tokenImgUrl`);
+                resetField(`actions.${actionIndex}.tokenSymbol`);
+                resetField(`actions.${actionIndex}.tokenBalance`);
+            }
+
+            return validationResult;
+        },
+        [actionIndex, provider, resetField]
+    );
+
+    const recipientValidator = useCallback(
+        async (value: InputValue) => {
+            const recipient = new Web3Address(provider, value.address, value.ensName);
+
+            // empty field
+            if (value.address === '' && value.ensName === '')
+                return t('errors.required.recipient');
+
+            // withdrawing to DAO
+            if (
+                recipient.address === daoDetails?.address ||
+                recipient.ensName === toDisplayEns(daoDetails?.ensDomain)
+            )
+                return 'Cant withdraw to your own address';
+
+            if (recipient.ensName && !recipient.address)
+                return (await recipient.isValidEnsName()) ? true : 'Invalid ENS name';
+
+            if (recipient.address && !recipient.ensName)
+                return recipient.isAddressValid() ? true : t('errors.invalidAddress');
+        },
+        [daoDetails?.address, daoDetails?.ensDomain, provider, t]
+    );
+
```

Comment:
Not needed we are not doing a withdrawn, just adding or removing addresses to a group